mysite/search/templates/search/results_widget.html

```diff
@@ -99,6 +99,23 @@
         <a href='#' id='expand-all-link'><img src={% version '/static/images/icons/zoom-in.png' %}>Expand all</a>
         <a href='#' id='collapse-all-link'><img src={% version '/static/images/icons/zoom-out.png' %}>Collapse all</a>
     </span>
+    <span class='active_facet_value'>
+        View:
+    </span>
+    <span id='item-limit'>
+        <form action="/search/?{{ query_str }}" method="post">
```

Comment:
I believe that @vipul-sharma20 had asked for a sanity-check on this.

I believe this is pretty reasonable. One thing I'd be slightly worried about is double-escaping; if it works, though, then great.